text/0032-minimal-relay.md

```diff
@@ -0,0 +1,190 @@
+1. The frozen state will form the genesis of a new system parachain.
+1. A Relay Chain runtime upgrade will allow migrating the deposit to the parachain. The parachain
+   deposit is on the order of 1/100th of the Relay Chain's. Therefore, this will result in freeing
+   up Relay State as well as most of each user's reserved balance.
+1. The pallet and any leftover state can be removed from the Relay Chain.
+
+User interfaces that render Identity information will need to source their data from the new system
+parachain.
+
+#### Staking
+
+Migrating the staking subsystem will likely be the most complex technical undertaking, as the
+Staking system cannot stop (the system MUST always have a validator set) nor run in parallel (the
+system MUST have only _one_ validator set) and the subsystem itself is made up of subsystems in the
+runtime and the node. For example, if offences are reported to the Staking parachain, validator
+nodes will need to submit their reports there.
+
+Handling balances also introduces complications. The same balance can be used for staking and
+governance. Ideally, all balances stay on Asset Hub, and only report "credits" to system chains like
+Staking and Governance. However, staking mutates balances by issuing new DOT on era changes and for
+rewards. Allowing DOT directly on the Staking parachain would simplify staking changes.
+
+Given the complexity, it would be pragmatic to include the Balances pallet in the Staking parachain
+in its first version. Any other systems that use overlapping locks, most notably governance, will
+need to recognise DOT held on both Asset Hub and the Staking parachain.
+
+There is more discussion about staking in a parachain in [Moving Staking off the Relay
+Chain](https://github.com/paritytech/polkadot-sdk/issues/491).
+
+#### Governance
+
+Migrating governance into a parachain will be less complicated than staking. Most of the primitives
+needed for the migration already exist. The Treasury supports spending assets on remote chains and
+collectives like the Polkadot Technical Fellowship already function in a parachain. That is, XCM
+already provides the ability to express system origins across chains.
+
+Therefore, actually moving the governance logic into a parachain will be simple. It can run in
+parallel with the Relay Chain's governance, which can be removed when the parachain has demonstrated
+sufficient functionality. It's possible that the Relay Chain maintain a Root-level emergency track
+for situations like [parachains
+halting](https://forum.polkadot.network/t/stalled-parachains-on-kusama-post-mortem/3998).
+
+The only complication arises from the fact that both Asset Hub and the Staking parachain will have
+DOT balances; therefore, the Governance chain will need to be able to credit users' voting power
+based on balances from both locations. This is not expected to be difficult to handle.
+
+## Drawbacks
+
+None at present.
```

Comment:
Increased complexity as existing dApps/wallets have to connect to multiple chains to maintain existing functionality.
Less compute power for staking due to pov limitation.

Reply:
I can add it, but this has always been part of building on Polkadot. Applications span multiple chains. That's one of the primary motivations for having shared security. And these dApps/wallets should be instantiating light clients anyway, not connecting to RPC providers.